src/main/java/io/github/bookcrawler/cache/AuthorsCache.java

```diff
@@ -0,0 +1,38 @@
+package io.github.bookcrawler.cache;
+
+import io.github.bookcrawler.entities.Author;
+import io.github.bookcrawler.repositories.AuthorsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class AuthorsCache {
+
+    @Autowired
+    private AuthorsRepository authorsRepository;
+
+    List<Author> authors = new ArrayList<>();
```

Comment:
You can use Optional probably to avoid camparing with null :) than You can check isEmpty()